clang/include/clang/Basic/arm_sve.td

```diff
@@ -2086,7 +2086,7 @@ let TargetGuard = "sve2p1|sme2" in {
   def SVCNTP_COUNT : SInst<"svcntp_{d}", "n}i", "QcQsQiQl", MergeNone, "aarch64_sve_cntp_{d}", [IsOverloadNone, IsStreamingCompatible], [ImmCheck<1, ImmCheck2_4_Mul2>]>;
 }
 
-let TargetGuard = "sve2p1,b16b16" in {
+let TargetGuard = "(sve2|sme2),b16b16" in {
```

Comment:
If we are adding for sme2 then we need to add the streaming compatible flag and also update the tests.

Reply:
Yeah, I'll add the `IsStreamingCompatible` flag.

What needs to be updated in tests?

Reply:
I do believe we should add:
__arm_streaming_compatible; 
in the tests.

svbfloat16_t test_svadd_bf16_m(svbool_t pg, svbfloat16_t op1, svbfloat16_t op2)   __arm_streaming_compatible
{
  return SVE_ACLE_FUNC(svadd, _bf16, _m)(pg, op1, op2);
}

Reply:
Done.